src/components/NamespacedPage/NamespacedPage.tsx

```diff
@@ -9,23 +8,21 @@ import SignupView from '../Signup/SignupView';
 
 import './NamespacedPage.scss';
 
-type NamespaceContextData = {
-  namespace: string;
-};
+type NamespaceContextData = [string, boolean];
```

Comment:
Does it have to be an array of string and boolean like that?

Reply:
No, It doesn't. It can be an object.

Reply:
Maybe having it as an object would be better in that case. Something like `{ namespace: string, loaded: boolean }`?